.github/workflows/conventional-commits-step-lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Conventional Commits Lint Step
+
+on:
+  workflow_call:
+    inputs:
+      LABELS:
+        required: true
+        type: string
+      CONFIG_FILE:
+        required: true
+        type: string
+        default: .commitlintrc
+
+jobs:
+  lint-commits:
+    name: Lint Commits
+    runs-on: ${{ fromJson(inputs.LABELS) }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v5
+        with:
+          configFile: ${{ inputs.CONFIG_FILE }}
+
+
+  lint-pr:
+    name: Lint PR
```

Comment:
qua prende in automatico il branch / codice? visto che prima hai fatto il checkout ma non qua